docs/sample_schema_package/mailman_messages/messages.py

```diff
@@ -82,6 +92,13 @@ def packages(self):
         """List of packages affected by the action that generated this message."""
         return []
 
+    def _get_avatar_from_from_header(self, from_header):
+        """Converts a From email header to an avatar."""
+        # Extract the username
+        addr = parseaddr(from_header)[1]
+        username = addr.split("@")[0]
+        return schema_utils.user_avatar_url(username)
+
```

Comment:
Hmm, previously it would have looked up an avatar by the full email address, now only by the username. Has anything changed underneath that this is fixing?

Reply:
Yes, the `user_avatar_url` function is expecting usernames, not email adresses, the previous call was wrong.